quinn-proto/src/crypto/rustls.rs

```diff
@@ -278,19 +282,23 @@ impl crypto::ServerConfig for rustls::ServerConfig {
         params: &TransportParameters,
     ) -> Box<dyn crypto::Session> {
         let version = interpret_version(version).unwrap();
+        let suite = InitialSuite::from_provider((*self).as_ref()).unwrap();
         Box::new(TlsSession {
             version,
             got_handshake_data: false,
             next_secrets: None,
             inner: rustls::quic::Connection::Server(
                 rustls::quic::ServerConnection::new(self, version, to_vec(params)).unwrap(),
             ),
+            suite,
         })
     }
 
     fn initial_keys(&self, version: u32, dst_cid: &ConnectionId) -> Result<Keys, CryptoError> {
         let version = interpret_version(version)?;
-        Ok(initial_keys(version, dst_cid, Side::Server))
+        // We validate that this works on configuration, so we can `unwrap()` here.
```

Comment:
I'm not sure what "on configuration" means here. Can we cite the responsible code more explicitly?